playwright/tests/editori-lomake-yleiset.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { test, expect, Page } from '@playwright/test'
+import { unsafeFoldOption } from '../playwright-utils'
+import {
+  getLomakkeenPoistamisenOsoite,
+  kirjauduVirkailijanNakymaan,
+  lisaaLomake,
+  teeJaOdotaLomakkeenTallennusta,
+} from '../playwright-ataru-utils'
+
+test.describe.configure({ mode: 'serial' })
+
+let lomakkeenAvain: string
+let lomakkeenId: number
+let page: Page
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage()
+
+  await kirjauduVirkailijanNakymaan(page)
+
+  await page.route('**/lomake-editori/api/tarjonta/haku**', async (route) => {
+    await route.fulfill({
+      json: [{ oid: '1.2.246.562.29.00000000000000009710', yhteishaku: true }],
+    })
+  })
+
+  await page.route(
+    '**/lomake-editori/api/tarjonta/haku/1.2.246.562.29.00000000000000009710',
+    async (route) => {
+      await route.fulfill({
+        json: { yhteishaku: true },
+      })
+    }
+  )
+
+  const lomake = await lisaaLomake(page)
+
+  lomakkeenAvain = unsafeFoldOption(lomake.lomakkeenAvain)
+  lomakkeenId = unsafeFoldOption(lomake.lomakkeenId)
+
+  await teeJaOdotaLomakkeenTallennusta(page, lomakkeenId, async () => {
+    // FIXME: Jos lomakkeen nimen syöttää liian aikaisin, automaattitallennus ei triggeröidy! https://jira.eduuni.fi/browse/OY-4642
```

Comment:
Tämä kommentti viittaa tähän tikettiin. Hypoteesi oli, ettei tässä tarvitsisi enää odottaa jotain timeouttia, vaan voisi vain odottaa, että spinneri menee piiloon. Varmista, että näin on.